### PR TITLE
4.30.0 release notes

### PR DESCRIPTION
**Pull request by melinath**

### CODE REVIEW

**Comment by melinath** [comment]
Test reruns: https://ci-oss.hashicorp.engineering/buildConfiguration/GoogleCloud_ProviderGoogleCloudGoogleProject/316321

**Comment by melinath** [comment]
Eliminated remaining failures as requiring any release changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 ## 4.30.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_service_account_jwt` ([#4489](https://github.com/hashicorp/terraform-provider-google-beta/pull/4489))
+* **New Resource:** `google_certificate_map_entry` ([#4501](https://github.com/hashicorp/terraform-provider-google-beta/pull/4501))
+* **New Resource:** `google_certificate_map` ([#4501](https://github.com/hashicorp/terraform-provider-google-beta/pull/4501))
+* **New Resource:** google_compute_backned_bucket_iam_binding ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```
Comment (megan07) [inline CHANGELOG.md:7]:
```suggestion
* **New Resource:** `google_compute_backend_bucket_iam_binding` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+* **New Resource:** google_compute_backned_bucket_iam_member ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```
Comment (megan07) [inline CHANGELOG.md:8]:
```suggestion
* **New Resource:** `google_compute_backend_bucket_iam_member` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+* **New Resource:** google_compute_backned_bucket_iam_policy ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```
Comment (megan07) [inline CHANGELOG.md:9]:
```suggestion
* **New Resource:** `google_compute_backend_bucket_iam_policy` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+* **New Resource:** google_dataproc_metastore_federation ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```
Comment (megan07) [inline CHANGELOG.md:10]:
```suggestion
* **New Resource:** `google_dataproc_metastore_federation` ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+* **New Resource:** google_dataproc_metastore_federation_iam_binding ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```
Comment (megan07) [inline CHANGELOG.md:11]:
```suggestion
* **New Resource:** `google_dataproc_metastore_federation_iam_binding` ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,33 @@
...
+* **New Resource:** google_dataproc_metastore_federation_iam_member ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```
Comment (megan07) [inline CHANGELOG.md:12]:
```suggestion
* **New Resource:** `google_dataproc_metastore_federation_iam_member` ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+* **New Resource:** google_dataproc_metastore_federation_iam_policy ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```
Comment (megan07) [inline CHANGELOG.md:13]:
```suggestion
* **New Resource:** `google_dataproc_metastore_federation_iam_policy` ([#4482](https://github.com/hashicorp/terraform-provider-google-beta/pull/4482))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,33 @@
 ## 4.30.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_service_account_jwt` ([#4489](https://github.com/hashicorp/terraform-provider-google-beta/pull/4489))
+* **New Resource:** `google_certificate_map_entry` ([#4501](https://github.com/hashicorp/terraform-provider-google-beta/pull/4501))
+* **New Resource:** `google_certificate_map` ([#4501](https://github.com/hashicorp/terraform-provider-google-beta/pull/4501))
+* **New Resource:** `google_compute_backend_bucket_iam_binding` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
+* **New Resource:** `google_compute_backned_bucket_iam_member` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```
Comment (megan07) [inline CHANGELOG.md:8]:
```suggestion
* **New Resource:** `google_compute_backend_bucket_iam_member` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,33 @@
...
+* **New Resource:** `google_compute_backned_bucket_iam_policy` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```
Comment (megan07) [inline CHANGELOG.md:9]:
```suggestion
* **New Resource:** `google_compute_backend_bucket_iam_policy` ([#4484](https://github.com/hashicorp/terraform-provider-google-beta/pull/4484))
```

File: CHANGELOG.md (hunk 8 above)
Comment (melinath) [reply CHANGELOG.md:8]: whoops, thanks!

**Review by megan07 — APPROVED** [review]

**Pull request merged**